tests/testcases/030_check-network.yml

```diff
@@ -54,7 +54,29 @@
     delay: 5
 
   - name: Run 2 busybox pods in test ns
-    command: "{{ bin_dir }}/kubectl run {{ item }} --image={{ test_image_repo }}:{{ test_image_tag }} --namespace test --command -- tail -f /dev/null"
+    shell:
+      cmd: |
+        cat <<EOF | {{ bin_dir }}/kubectl apply -f -
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          name: {{ item }}
+          namespace: test
+        spec:
+          containers:
+          - name: busybox
+            image: {{ test_image_repo }}:{{ test_image_tag }}
+            command: ['tail', '-f', '/dev/null']
+            securityContext:
+              allowPrivilegeEscalation: false
+              capabilities:
+                add: ['CAP_NET_RAW']
```

Comment:
This capability is for the above issue.

Reply:
```
TASK [Run 2 busybox pods in test ns] *******************************************
task path: /builds/kargo-ci/kubernetes-sigs-kubespray/tests/testcases/030_check-network.yml:56
Thursday 13 October 2022  02:07:50 +0000 (0:00:00.620)       0:00:05.724 ****** 
failed: [instance-1] (item=busybox1) => {"ansible_loop_var": "item", "changed": false, "cmd": "cat <<EOF | /usr/local/bin/kubectl apply -f -\napiVersion: v1\nkind: Pod\nmetadata:\n  name: busybox1\n  namespace: test\nspec:\n  containers:\n  - name: busybox\n    image: registry.k8s.io/busybox:latest\n    command: ['tail', '-f', '/dev/null']\n    securityContext:\n      allowPrivilegeEscalation: false\n      capabilities:\n        add: ['CAP_NET_RAW']\n        drop: ['ALL']\n      runAsUser: 1000\n      runAsNonRoot: true\n      seccompProfile:\n        type: RuntimeDefault\nEOF\n", "delta": "0:00:00.824175", "end": "2022-10-13 02:07:52.042868", "item": "busybox1", "msg": "non-zero return code", "rc": 1, "start": "2022-10-13 02:07:51.218693", "stderr": "Error from server (Forbidden): error when creating \"STDIN\": pods \"busybox1\" is forbidden: violates PodSecurity \"restricted:latest\": unrestricted capabilities (container \"busybox\" must not include \"CAP_NET_RAW\" in securityContext.capabilities.add)", "stderr_lines": ["Error from server (Forbidden): error when creating \"STDIN\": pods \"busybox1\" is forbidden: violates PodSecurity \"restricted:latest\": unrestricted capabilities (container \"busybox\" must not include \"CAP_NET_RAW\" in securityContext.capabilities.add)"], "stdout": "", "stdout_lines": []}
```

Reply:
hum, the above capability is invalid against `kube_pod_security_default_enforce: restricted`